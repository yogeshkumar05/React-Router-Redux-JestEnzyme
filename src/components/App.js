import React, {Component} from 'react';
import Navbar from './Navbar';

export default class App extends Component
{
    render()
    {
        return(
            <Navbar/>
        )
    }
}