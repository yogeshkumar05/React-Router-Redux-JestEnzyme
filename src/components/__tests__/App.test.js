import {shallow} from 'enzyme';
import App from '../App';
import React from 'react';

describe('App Component', () =>{
    it('snapshot test', () => {
        const wrapper = shallow(<App/>);
        expect(wrapper).toMatchSnapshot();
    });
})