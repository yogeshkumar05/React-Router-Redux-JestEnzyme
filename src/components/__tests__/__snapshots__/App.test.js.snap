// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component snapshot test 1`] = `ShallowWrapper {}`;
